src/pages/stacks-blockchain/transactions.md

```diff
@@ -0,0 +1,462 @@
+  "tx_status": "success",
+  "tx_type": "token_transfer",
+  "fee_rate": "180",
+  "sender_address": "STJTXEJPJPPVDNA9B052NSRRBGQCFNKVS178VGH1",
+  "sponsored": false,
+  "post_condition_mode": "deny",
+  "block_hash": "0x9080f6df3e0be0d6de67569330e547346a44c8ecd30d9d76b5edd1b49e2c22f6",
+  "block_height": 3190,
+  "burn_block_time": 1594227992,
+  "canonical": true,
+  "tx_index": 1,
+  "token_transfer": {
+    "recipient_address": "ST1RZG804V6Y0N4XHQD3ZE2GE3XSCV3VHRKMA3GB0",
+    "amount": "10000",
+    "memo": "0x00000000000000000000000000000000000000000000000000000000000000000000"
+  },
+  "events": [
+    {
+      "event_index": 0,
+      "event_type": "stx_asset",
+      "asset": {
+        "asset_event_type": "transfer",
+        "sender": "STJTXEJPJPPVDNA9B052NSRRBGQCFNKVS178VGH1",
+        "recipient": "ST1RZG804V6Y0N4XHQD3ZE2GE3XSCV3VHRKMA3GB0",
+        "amount": "10000"
+      }
+    }
+  ]
+}
+```
+
+## Signature and Verification
+
+## Broadcast
+
+With a serialized transaction in the [raw format](#raw-format), it can be broadcasted to the network using the [`POST /v2/transactions`](https://blockstack.github.io/stacks-blockchain-api/#operation/post_core_node_transactions) endpoint:
+
+```shell
+curl --location --request POST 'https://stacks-node-api-latest.argon.blockstack.xyz/v2/transactions' \
+--header 'Content-Type: text/plain' \
+--data-raw '<tx_raw_format>'
+```
+
+The API will respond with a `HTTP 200 - OK` if the transactions was successfully added to the mempool.
+
+## Querying
+
+Transactions on the Stacks 2.0 network can be queried using the [Stacks Blockchain API](/references/stacks-blockchain). The API exposes two interfaces, a RESTful JSON API and a WebSockets API.
+
+For convenience, a Postman Collection was created and published: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/614feab5c108d292bffa)
+
+-> Note: The API can be easily consumed using a generated [JS client library](https://blockstack.github.io/stacks-blockchain-api/client/index.html). The generator uses an OpenAPI specification and supports other languages and frameworks.
+
+### Pagination
+
+To make API responses more compact, lists returned by the API are paginated. For lists, the response body includes:
+
+- `limit`: the number of list items return per response (max is `200`)
+- `offset`: the number of elements to skip (starting from `0`)
+- `total`: the number of all available list items
+- `results`: the array of list items (length of array is between 0 and the set limit)
+
+Using the `limit` and `offset` properties, you can paginate through the entire list by increasing the offset by the limit until you reach the end of the list (as indicated by the `total` field).
+
+### Filter
+
+The current API design doesn't allow to set filters for queries. In case you are looking for a specific type of transaction, it is best to paginate through the list and iterate over the results of each response.
```

Comment:
Reading this a reader might naturally ask "why"? Any context we can add on why filters are not supported, or if they are planned in future?

Reply:
Checking with @zone117x to make a callout for this. thanks!

Reply:
update: there's actually API capability to filter - it is not yet documented. will fix and update this content!